src/reducers/taggable.js

```diff
@@ -52,6 +53,20 @@ export default function taggable (state = initialState, action) {
         ...state,
         configurationSaved: true
       };
+    case DELETE_VALUE:
```

Comment:
If you can, try and think of a way to make this cleaner! I'm not a fan of mutating arrays but I can't think of anything better at the moment!